src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -874,8 +870,7 @@ private IList<CodeFlow> CreateCodeFlows(ContrastLogReader.Context context)
                          {
                              new ThreadFlow
                              {
-                                  Locations = context.PropagationEvents,
-                                  ImmutableState = context.Headers
```

Comment:
>Headers [](start = 59, length = 7)

Moved to the new request object.

Reply:
yay! :)

---
In reply to: [274063050](https://github.com/Microsoft/sarif-sdk/pull/1421#discussion_r274063050) [](ancestors = 274063050)